### ISSUE TITLE
User 'routinator' already exists on RPM re-install

### ISSUE THREAD

**Issue by ximon18** [NLnetLabs/routinator#774]
Platform: Fedora 36 x86_64

Having had the Routinator RPM installed and then uninstalled, a new install reports an error:

```
$ sudo dnf install ./routinator-0.12.0-dev-1.x86_64.rpm
Last metadata expiration check: 4:56:18 ago on Tue 23 Aug 2022 17:08:27 CEST.
Dependencies resolved.
==============================================================================================================================================================================================================================================
 Package                                                  Architecture                                         Version                                                       Repository                                                  Size
==============================================================================================================================================================================================================================================
Installing:
 routinator                                               x86_64                                               0.12.0-dev-1                                                  @commandline                                               2.7 M

Transaction Summary
==============================================================================================================================================================================================================================================
Install  1 Package

Total size: 2.7 M
Installed size: 8.6 M
Is this ok [y/N]: y
Downloading Packages:
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                                                                                                                                      1/1 
  Installing       : routinator-0.12.0-dev-1.x86_64                                                                                                                                                                                       1/1 
  Running scriptlet: routinator-0.12.0-dev-1.x86_64                                                                                                                                                                                       1/1 
useradd: user 'routinator' already exists
warning: %post(routinator-0:0.12.0-dev-1.x86_64) scriptlet failed, exit status 9

Error in POSTIN scriptlet in rpm package routinator
  Verifying        : routinator-0.12.0-dev-1.x86_64                                                                                                                                                                                       1/1 

Installed:
  routinator-0.12.0-dev-1.x86_64                                                                                                                                                                                                              

Complete!
```

Notice the errors:
```
useradd: user 'routinator' already exists
warning: %post(routinator-0:0.12.0-dev-1.x86_64) scriptlet failed, exit status 9

Error in POSTIN scriptlet in rpm package routinator
```

**Comment by ximon18** [comment]
If I recall correctly the guidance is to leave created user accounts behind as they may still own files, so we should accept an existing user account.

**Comment by ximon18** [comment]
Fixed by PR #780.
